digits/views.py

```diff
@@ -435,6 +436,11 @@ def path_autocomplete():
 
     """
     path = flask.request.args.get('query','')
+
+    if not os.path.isabs(path) :
+        # Only allow absolute paths by appending forward slash
+        path = '/' + path
```

Comment:
@IsaacYangSLA can you try this on Windows? Maybe something like this would be better:
```py
path = os.path.sep + path
```